lesson3/exercise-two.js

```diff
@@ -0,0 +1,5 @@
+let breakfastqt =['Bacon', 'and','eggs','are', 'delicious!'];
+
+const breakfastqtbrand = breakfastqt.join(' ');
+console.log(breakfastqtbrand);
+console.log('eggs')
```

Comment:
Yes, this is technically accurate, but I'm hoping you can use the array[] syntax to target the 3rd word